sdk/src/config/MAINNET.ts

```diff
@@ -122,6 +123,16 @@ const MAINNET: { [chain in MainnetChainName]: ChainConfig } = {
     contracts: CONTRACTS.MAINNET.aptos,
     finalityThreshold: 0,
   },
+  sei: {
+    key: 'sei',
+    id: 32,
+    context: Context.SEI,
+    contracts: {
+      ...CONTRACTS.MAINNET.sei,
+      seiTokenTranslator: '',
+    },
+    finalityThreshold: 0,
+  },
```

Comment:
Just comment out this `sei` block as well as any sei tokens.  That should remove it completely from the ui, we can uncomment when we are ready for mainnet

Reply:
Commented the sei related entries on the `mainnet.ts` file